src/Components/ApplicationFormModal.js

```diff
@@ -25,33 +25,52 @@ const CREATE_APPLICATION = gql`
             }
 `
 
-const ApplicationFormModal = ({ show, onClose, petName, petId, refetch }) => {
+const ApplicationFormModal = ({ show, onClose, petName, petId }) => {
     const [formState, setFormState] = useState({
         name: '',
         email: '',
         description: ''
     })
 
-    const [createApplication, { data, loading, error}] = useMutation(CREATE_APPLICATION)
+    const [createApplication] = useMutation(CREATE_APPLICATION, {
+      refetchQueries: [
```

Comment:
MAGIC!!! Thank you Apollo Client for this being a thing. Well done Ethan on finding this really smooth functionality.